news/management/commands/ingest_news.py

```
@@ -123,17 +141,52 @@ def _process_entry(self, source, entry):
+                article.content = doc.summary() # This is cleaned HTML
+
+                # If we still don't have a thumbnail, try to get the first image from the scraped content
+                if not article.image_url:
+                    img_match = re.search(r'<img [^>]*src="([^"]+)"', article.content)
+                    if img_match:
+                        # Make sure URL is absolute
+                        article.image_url = urljoin(article.url, img_match.group(1))
+
+                article.save()
+                self.stdout.write(self.style.SUCCESS(f"  ✓ Scraped and updated: {article.title[:60]}..."))
+             except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)